docs/user/adding-glean-to-your-project.md

```diff
@@ -45,8 +46,39 @@ For example, if version *0.34.2* is used, then the include directive becomes:
 apply from: 'https://github.com/mozilla/glean/raw/v0.34.2/glean-core/android/sdk_generator.gradle'
 ```
 
+### Adding new metrics
+
+All metrics that your project collects must be defined in a `metrics.yaml` file. 
+This file should be at the root of the module (the same directory as the `build.gradle` file you updated). 
+The format of that file is documented [here](https://mozilla.github.io/glean_parser/metrics-yaml.html).
+To learn more, see [adding new metrics](adding-new-metrics.md).
+
+**Important**: as stated [here](#before-using-the-library), any new data collection requires documentation and data-review. This is also required for any new metric automatically collected by Glean.
```

Comment:
`here`.
IMO `here` or maybe `above` would be a sensible alternative (given that it is a link in the same document)

Reply:
Is there any way we could have rendered version of the docs display these "important" sections as we do in [the desktop docs](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/start/adding-a-new-probe.html)? Not as part of this bug, but it might be worthwhile doing in a separate bug.

Reply:
We can: ![important quoted block stands out](https://tmp.fnordig.de/scr/60b1112cdb.png)

```
> **Important**: as stated [here](#before-using-the-library), any new data collection, including new custom pings, requires documentation and data-review. This is also required for any new ping automatically collected by Glean.
```

Reply:
NEAT!!!